docs/podman-replay-kube.1.md

```diff
@@ -0,0 +1,70 @@
+% podman-generate Podman Man Pages
+% Brent Baude
+% December 2018
+# NAME
+podman-replay-kube - Recreate pods and containers based on Kubernetes YAML
+
+# SYNOPSIS
+**podman replay kube **
+[**-h**|**--help**]
+kubernetes_input.yml
+
+# DESCRIPTION
+**podman replay kube** will read in a structured file of Kubernetes YAML.  It will then recreate
```

Comment:
Can you talk about how the file would be generated or it's format?  Do we need a man page for the file itself?  IDK about that last bit, throwing it up for consideration.